src/kiwixapp.cpp

```diff
@@ -403,8 +403,7 @@ void KiwixApp::createActions()
     HIDE_ACTION(SavePageAsAction);
     */
 
-    CREATE_ACTION_SHORTCUT(SearchArticleAction, gt("search-article"), QKeySequence(Qt::CTRL | Qt::Key_L));
-    HIDE_ACTION(SearchArticleAction);
+    CREATE_ACTION_SHORTCUTS(SearchArticleAction, gt("search-article"), QList<QKeySequence>({QKeySequence(Qt::Key_F6), QKeySequence(Qt::CTRL | Qt::Key_L), QKeySequence(Qt::ALT | Qt::Key_D)}));
```

Comment:
Did you unhide the `SearchArticleAction` intentionally?

Reply:
Yeah, the last commit history says actions not needed for the alpha are hidden and is 6 years ago. Since #436 wanted this feature, I believe it is ok to revive this feature?